Machine_Learning/Handwritten Text Recognition Model/requirements.txt

```diff
@@ -0,0 +1,13 @@
+easyocr==1.7.1
+googletrans==4.0.0rc1
+imutils==0.5.4
+langdetect==1.0.9
+numpy==2.1.2
+opencv_contrib_python==4.10.0.84
+opencv_python==4.10.0.82
+opencv_python_headless==4.10.0.84
+Pillow==11.0.0
+scipy==1.11.4
+streamlit==1.30.0
+streamlit==1.36.0
```

Comment:
Hey @Sameeratweb ,
Make sure there is no clash because of 2 different versions of same thing (in this case streamline). Also try to use more stable versions like for `numpy` use `1.24.x` and for `Pillow` use `10.x.x`. Always make sure these are compatible with each other.

For future use, try using `pip freeze > requirements.txt`  to generate `requirements.txt`. It will take care of the compatibility.

The program works fine, please update the changes and I will approve.
Thank you!

Reply:
streamlit==1.30.0
streamlit==1.36.0